### PR TITLE
Todo list best practice

### PR DESCRIPTION
**Pull request by zewdu444**
### This project used for  to asking code review from collaborator  specifically  on best practice of HTML,CSS, and Javascript

- [x] Project were cloned from Todo list application.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <title>Todo list app</title>
+  </head>
+  <body>
+    <main>
+      <div class="titleholder">
+        <p>Today's To Do</p>
+        <p><i class="fa-solid fa-arrows-rotate"></i></p>
+      </div>
+      <hr />
+      <div class="inputholder">
```
Comment (Tobby8629) [inline src/index.html]: Tag worked perfectly okay but using forms would have been better 👍

File: src/module/deleteTask.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import tasks from './taskStore.js';
+
+const deleteTask = (index) => {
+  const temp = tasks.filter((task) => task.index !== index);
+  tasks.length = 0;
+  let counter = 1;
+  temp.forEach((element) => {
+    element.index = counter;
```
Comment (Tobby8629) [inline src/module/deleteTask.js:8]: Nice work, but you could assign each element to its index by using index as the second parameter in the call back function

File: src/module/updateTask.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const updateTask = (index, description) => {
+  tasks.filter((object) => object.index === index).forEach((object) => {
+    object.description = description;
+  });
+};
+
+const completeTask = (index, completed) => {
+  tasks.filter((object) => object.index === index).forEach((object) => {
+    object.completed = completed;
```
Comment (Tobby8629) [inline src/module/updateTask.js:11]: According to the requirement completed should set to Boolean value

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  margin-right: auto;
+  margin-top: 10%;
+  max-width: 375px;
+}
+
+hr {
+  width: 99.9%;
+  border: 0.05px solid #edeaea;
+  color: #edeaea;
+  margin-bottom: 0;
```
Comment (Tobby8629) [inline src/style.css:22]: To avoid repetition you can easily use
margin : 0 auto
The 0 for margin up and down 
The auto for left and right

**Review by Tobby8629 — COMMENTED** [review]
Nice job 👍 
- neat codes
- Proper variable names 
- Code super simple
- few DRY
But there are few changes 🤏🏽 you can make to make your code work and look better

File: src/module/deleteTask.js (hunk 2 above)
Comment (zewdu444) [reply src/module/deleteTask.js:8]: - Thank you for your feedback. If you use as a second parameter, it is difficult to begin indexing at 1.

File: src/index.html (hunk 1 above)
Comment (zewdu444) [reply src/index.html]: - Okay, thank you; I switched to form element.

File: src/module/updateTask.js (hunk 3 above)
Comment (zewdu444) [reply src/module/updateTask.js:11]: - Please check on the implementation part in  index.js file it accept only Boolean value

File: src/style.css (hunk 4 above)
Comment (zewdu444) [reply src/style.css:22]: Thanks remove all repeated margin parameter accordingly.

**Pull request merged**